packages/language-server/src/plugins/typescript/module-loader.ts

```diff
@@ -61,16 +63,27 @@ class ModuleResolutionCache {
         const fileNameWithoutEnding =
             getLastPartOfPath(this.getCanonicalFileName(path)).split('.').shift() || '';
         this.cache.forEach((val, key) => {
+            const split = key.split(':::');
```

Comment:
`moduleName` and `containingFile` retrieval can probably be combined, right now they both split.